pkg/scheduler/core/scheduling_queue.go

```diff
@@ -410,11 +410,9 @@ func (p *PriorityQueue) AssignedPodUpdated(pod *v1.Pod) {
 func (p *PriorityQueue) MoveAllToActiveQueue() {
 	p.lock.Lock()
 	defer p.lock.Unlock()
-	var unschedulablePods []interface{}
 	for _, pod := range p.unschedulableQ.pods {
-		unschedulablePods = append(unschedulablePods, pod)
+		p.activeQ.Add(pod)
```

Comment:
Should we log error message if return error?

Reply:
The `Add()` function logs the error. We don't need to log again here.

Reply:
`Heap.Add` just return error, `PriorityQueue.Add` logged error; `unschedulableQ` and `activeQ` are using different keyFunc :)